pylops/linearoperator.py

```diff
@@ -332,9 +346,14 @@ def __rmul__(self, x: float) -> LinearOperator:
         else:
             return NotImplemented
 
+    def __rmatmul__(self, other):
+        if np.isscalar(other):
+            raise ValueError("Scalar not allowed, use * instead")
+        return self.__rmul__(other)
+
     def __pow__(self, p: int) -> LinearOperator:
         if np.isscalar(p):
-            Op = aslinearoperator(_PowerLinearOperator(self, p))
```

Comment:
Now that we implement all private methods, there is no more need for this...

This comment applies in many places :)

Reply:
Noted and agreed